elasticproxy/src/main/java/zone/cogni/asquare/elasticproxy/ElasticsearchProxyTemplate.java

```diff
@@ -191,7 +191,16 @@ private ResponseEntity<Resource> executeRequest(String requestUUID, HttpMethod h
     Request request = new Request(httpMethod.name(), buildUrl(spel(url, params), urlParams));
 
     if (HttpMethod.POST.equals(httpMethod) || HttpMethod.PUT.equals(httpMethod)) {
-      request.setJsonEntity(thymeleaf(getTemplateContent(), params));
+      String templatedBody = thymeleaf(getTemplateContent(), params);
+
+      if (templatedBody == null) {
```

Comment:
what if we skip the null and instead catch the exception here, rather than in the other method? then this logic can be in the catch here 🤷 

And then it could be in the same try/catch as the one below. And since it looks like we need a separate treatment for this one (compared to the existing that will put the thymeleaf error message in the response), have 2 catch clauses?

Reply:
I actually implemented it like this at first but then changed it because I felt that the exception should be catched asap. I'll change it :)